### DISCUSSION TITLE
Cursor ripple (i assume) is too big on one of my skins

### DISCUSSION THREAD

**Discussion by Dementedduck** [ppy/osu#26392] [Q&A] [answered]
I wanted to try lazer so i loaded the same skin i use in stable and saw this https://streamable.com/gylq2v
i assume its not a feature for obvious reasons but it goes away completely when turning off cursor ripple ... the problem is i like ripple ;-;
i dunno what this could be related to in the skin but I assume it should be an easy fix if it can be found

heres the skin if curious: https://drive.google.com/file/d/1Fq7RToHizSGmOzla0AmG1ud7ypybU_HO/view?usp=sharing

**Comment by nagi-desuuu** [comment, marked as answer]
https://github.com/ppy/osu/issues/23870

**Reply by nagi-desuuu** [reply]
Sorta related: https://github.com/ppy/osu/issues/23099#issuecomment-1704024641
